.github/workflows/platform.yml

```diff
@@ -35,6 +35,18 @@ jobs:
       - run: ./node_modules/.bin/bun pm pack --gzip-level 0 --filename bun.tgz && tar -zxf bun.tgz -C bun
       - run: diff -r npm bun
 
+  bundle-validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 # v6.0.0
+      - uses: ./.github/actions/node/active-lts
+      # Running `yarn` also automatically runs `rspack` as a postinstall script.
+      - run: yarn --frozen-lockfile
+        working-directory: vendor
+      # Licenses are pulled from internet and not deterministic so revert them.
+      - run: git checkout packages/node_modules/**/*/LICENSE
```

Comment:
I don't understand this step. Are we on purpose reverting any license change that the vendored dependency have made to their own license, and if so why?